eclipse-collections-testutils/src/main/java/org/eclipse/collections/impl/test/ClassComparer.java

```diff
@@ -179,6 +192,12 @@ public MutableSortedSet<String> getMethodNames(Class<?> classOne)
         return ArrayIterate.collectIf(classOne.getMethods(), this::includeMethod, this::methodName, SortedSets.mutable.empty());
     }
 
+    public MutableSortedSet<String> getInstanceMethodNames(Class<?> classOne)
+    {
+        Method[] staticMethods = Arrays.stream(classOne.getMethods()).filter(method -> Modifier.isStatic(method.getModifiers())).toArray(Method[]::new);
```

Comment:
I think should either use reject from EC or use filter( !Modifier.isStatic). It would then make more sense for the variable to be named instanceMethods instead of staticMethods.